scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/yaml/AbstractYAMLOpInfoCreator.java

```diff
@@ -16,24 +28,35 @@ public abstract class AbstractYAMLOpInfoCreator implements YAMLOpInfoCreator {
 
     @Override
     public OpInfo create(final URI identifier, final Map<String, Object> yaml) {
-        // Parse path - start after the leading slash
-        final String path = identifier.getPath().substring(1);
-        // Parse source
-        final String srcString = path.substring(0, path.indexOf('/'));
-        // Parse version
-        final String version = path.substring(path.indexOf('/') + 1);
+        // Parse source - start after the leading slash
+        final String srcString = identifier.getPath().substring(1);
+        // TODO: Parse version
+        final String version = "0.0";
+//        final String version = path.substring(path.indexOf('/') + 1);
+        final Map<String, Object> tags = subMap(yaml, "tags");
         // Parse names
         final String[] names;
-        if (yaml.containsKey("name")) {
-            names = new String[]{(String) yaml.get("name")};
+        if (tags.containsKey("name")) {
+            names = new String[]{(String) tags.get("name")};
         } else {
-            String namesString = (String) yaml.get("names");
-            names = namesString.split("\\s*,\\s*");
+            var tmp = tags.get("names");
```

Comment:
Should also consider the case where there *is* no name - probably should throw an exception there, too.

Reply:
We do now!